app/views/games/show.html.erb

```diff
@@ -1,3 +1,31 @@
+<script type="text/javascript">
+  $(function() {
+    $('.piece').draggable({
+      cursor: 'crosshair'
+    });
+
+    $('.left').droppable({
+      drop: function(event, ui) {
+          alert("X position = " + ui.draggable.data('x') + "  Y position = " + ui.draggable.data('y'))
+          $.ajax({
+            type: 'PUT',
+            url: ui.draggable.data('update-game'),
```

Comment:
this should be the `pieces/:id`

`piece_path(piece)`

You're updating the x/y for a specific piece. So you want to hit the route for it.

pieces_controller#update